pex/util.py

```diff
@@ -41,6 +41,32 @@ def zipsafe(dist):
       return False
 
   @classmethod
+  def access_zipped_assets(cls, static_module_name, static_path, asset_path, dir_location='.'):
+    """
+    Create a copy of static resource files as we can't serve
+    them from within the pex file.
+
+    :param static_module_name: Module name containing module to cache in a tempdir
+    :type static_module_name: string in the form of 'twitter.common.zookeeper'
+    :param static_path: Module name of the form 'serverset'
+    :param asset_path: Initially a module name that's the same as the static_path, but will be
+        changed to walk the directory tree
+    :param dir_location: directory to create a new temporary directory in
+    """
+    temp_dir = safe_mkdtemp(dir=dir_location)
+    for asset in resource_listdir(static_module_name, asset_path):
+      asset_target = os.path.join(os.path.relpath(asset_path, static_path), asset)[2:]
```

Comment:
It's gotta remote a `./` which prepends the file name. It looks like this before/after the `[2:]`
```
./mod.py
mod.py
./mod.pyc
mod.pyc
./__init__.py
__init__.py
```

I've swapped this to use `os.path.normpath` to get what I'm actually looking for here.

Happy for other suggestions though.